DittoExportLogs/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     implementation libs.androidx.compose.material3.material3
     implementation libs.androidx.compose.ui.ui
     implementation libs.androidx.navigation.navigationCompose
+    implementation(libs.live.ditto.ditto)
```

Comment:
You'll want to merge the latest main as it resolves some of these dependency issues, specifically with the ditto sdk